src/AvTranscoder/properties/FileProperties.cpp

```diff
@@ -31,10 +31,6 @@ void FileProperties::extractStreamProperties(IProgress& progress, const EAnalyse
 {
     clearStreamProperties();
 
-    // if the analysis level wiil decode some streams parts, seek at the beginning
-    if(level > eAnalyseLevelHeader)
-        const_cast<FormatContext*>(_formatContext)->seek(0, AVSEEK_FLAG_BACKWARD);
-
```

Comment:
Could you explain why seeking at the beginning of the stream " _make analysis impossible for some AVC Intra media_ " ?